src/NetAnalyzers/Core/Microsoft.CodeQuality.Analyzers/ApiDesignGuidelines/TypesThatOwnDisposableFieldsShouldBeDisposable.cs

```diff
@@ -32,6 +32,8 @@ public abstract class TypesThatOwnDisposableFieldsShouldBeDisposableAnalyzer<TTy
             isPortedFxCopRule: true,
             isDataflowRule: false);
 
+        private readonly List<string> _disposableFieldNames = new();
```

Comment:
❗ Analyzers are not allowed to store state directly on the instance. This will need to be declared inside `AnalyzeSymbolStart`.

Reply:
Done.